skeleton/techdoc/docs/rhoai.md

```diff
@@ -0,0 +1,35 @@
+# Running Samples in OpenShift AI
+
+This document will outline how you can run build and run your sample applications within an OpenShfit AI workbench.
+
+## Prerequisites
+
+- `oc` cli installed
```

Comment:
Should we link to `oc` install page here?

Reply:
Yeah, that's a good idea. I'll link https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-mac.tar.gz which will provide the latest stable binary

Reply:
Updated